docs/source/release_notes/release-notes.rst

```diff
@@ -46,6 +46,12 @@ develop
          - Any lock requests that take more than ``100ms`` to receive a response are now logged in the ``SlowLockLogger`` logger.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1769>`__)
 
+    *    - |fixed|
+         - AtlasDB HTTP clients now parse ``Retry-After`` headers correctly.
+           This manifests as clients failing over and trying other nodes when receiving a 503 with a ``Retry-After`` header from a remote (e.g. from a TimeLock non-leader).
```

Comment:
Yes. Effectively, we ignored `Retry-After` headers (we always parsed them as being null). So if you don't have a header, the behaviour is unchanged (we'll attempt to parse it, and it'll be null). If you do, however, then we will respond to it, and retry elsewhere.

Actually, if you receive a 503 without a `Retry-After` header, you would previously retry 3x on the same node _and then_ fail over. This has been preserved, so that internal use case will be unaffected.